src/main/java/core/basesyntax/StoreOperations.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.function.DoubleBinaryOperator;
+
+public class StoreOperations {
+    static final HashMap<String, DoubleBinaryOperator> AVAILABLE_OPERATIONS = new HashMap<>();
```

Comment:
why Double? we use Int everywhere, so i suggest we use IntBinaryOperator, not to cast double to int in future

Reply:
done